docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+export GENESIS_DIR="${GENESIR_DIR:-/genesis}"
+export OUTPUT_DIR="${OUTPUT_DIR:-/output}"
+
+set -ex
+echo "Performing build of Genesis Framework"
+cd $GENESIS_DIR
+
```

Comment:
Interesting, I didn't think we published them to rubygems, just our local gem server.